src/stores/eth.js

```diff
@@ -26,7 +26,7 @@ const providerOptions = {
   walletconnect: {
     package: WalletConnectProvider, // required
     options: {
-      infuraId: 'e106b2b27c0f4941be1f2c183a20b3ea', // required
+      infuraId: '1ec103a49691457aa6dff30aa8ab73d0', // e106b2b27c0f4941be1f2c183a20b3ea
```

Comment:
this has to change

Reply:
yeah i know, just left it there to let you test.
i'm changing it :)